src/services/asterPlotConverterService.ts

```diff
@@ -247,9 +250,11 @@ export class AsterPlotConverterService {
                     tooltipInfo = this.buildOneMeasureTooltip(formattedCategoryValue, currentValue, localizationManager);
                 }
 
+                const height: number = isNotNull ? highlightValues[i] : null;
+                const width: number = Math.max(0, (categoricalColumns.Y.length > 1 && categoricalColumns.Y[1].highlights[i] !== null) ? <number>categoricalColumns.Y[1].highlights[i] : sliceWidth)
```

Comment:
seems we can create a variable for `<number>categoricalColumns.Y[1].highlights[i]` as it used multiple times here and above.